src/serialize.h

```diff
@@ -152,12 +152,11 @@ inline float ser_uint32_to_float(uint32_t y)
 // i.e. anything that supports .read(char*, size_t) and .write(char*, size_t)
 //
 
-enum
+enum SerializeType
```

Comment:
replace this with "enum SerializeType : int" in order to make absolutely sure there is no consensus code changes.